index.html

```diff
@@ -916,7 +916,16 @@ <h3>Concrete Lifecycle Example</h3>
     <span class='comment'>// identifier for the only subject of the credential</span>
     "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
     <span class='comment'>// assertion about the only subject of the credential</span>
-    "alumniOf": "&lt;span lang='en'>Example University&lt;/span>"
+    "alumniOf": {
+      "id": "did:example:c276e12ec21ebfeb1f712ebc6f1",
+      "name": [{
+        "value": "Example University",
+        "language": "en"
```

Comment:
These examples use the alias "language" instead of "lang". Which should it be?

Reply:
Replaced all "language" with "lang".